docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'setprchsreq.a'
+order: 32
+deploy: ['experimental']
+---
+
+# setprchsreq.a
+
+This action is used to set purchase requirements for a token factory.
+
+## Technical Behavior
+
+Factory manager can specify purchase options for users. Note that currently have to use the same action for both creation and modification of purchase requirements.
+
+-   If **asset_manager** is other than `ultra.nft.ft`, The cost of a factory creation is paid to `eosio.nftram` and it will be locked up in the purchase option.
```

Comment:
- If **asset_manager** is an account other than `ultra.nft.ft`, the cost of a factory creation is paid to `eosio.nftram` and it will be locked up in the purchase option.